### PR TITLE
Removed unreachable code

### PR DESCRIPTION
**Pull request by JoPintoPaul**
The setting of `isConditional` was removed in an earlier commit, so this class will never be set.

### CODE REVIEW

**Review by domoscargin — APPROVED** [review]
Thanks @JoPintoPaul Nice catch.

I'm just going to wait for somebody else on my team to approve before merging.

**Review by owenatgov — APPROVED** [review]
Looks good to me 👍🏻  Good spot @JoPintoPaul

**Pull request merged**
